databrowser/src/layouts/menu/DatasetMenuSubList.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <div class="flex flex-col w-full">
+    <span class="py-2 pl-2 text-xl font-semibold">{{ title }}</span>
+    <ul>
+      <li
+        v-for="item in items"
+        :key="item.label"
+        :class="[
+          selected == item.label
+            ? 'bg-green-500 bg-opacity-10'
+            : 'hover:bg-gray-50',
+        ]"
+        class="flex whitespace-nowrap"
+      >
+        <button
+          v-if="item.categories && item.categories.length"
+          class="
+            inline-flex
+            flex-1
+            justify-between
+            items-center
+            py-2
+            px-2
+            w-full
+            max-w-xs
+            text-left
+          "
+          @click="setSelected(item)"
+        >
+          <span class="pr-24">{{ item.label }}</span>
+          <ArrowRight />
+        </button>
+        <router-link
+          v-if="item.url"
+          :to="item.url"
+          class="flex-1 py-2 pr-24 pl-2 w-full text-left"
+          >{{ item.label }}
+        </router-link>
+      </li>
```

Comment:
The background color must go all the way from side to side.

<img width="423" alt="Bildschirmfoto 2022-01-12 um 13 21 53" src="https://user-images.githubusercontent.com/3337260/149139560-a145b6e2-6886-43dd-a196-22398216ae2d.png">